sql/core/src/test/resources/sql-tests/results/describe.sql.out

```diff
@@ -1,205 +1,248 @@
+Type                	MANAGED             	                    
+Provider            	parquet             	                    
+Num Buckets         	2                   	                    
+Bucket Columns      	[`a`]               	                    
+Sort Columns        	[`b`]               	                    
+Comment             	table_comment       	                    
+Location[removed by SQLQueryTestSuite]sql/core/spark-warehouse/t	                    
+Partition Provider  	Catalog             	                    
+Partition Columns   	[`c`, `d`]          	                    
+Schema              	root
+ |-- a: string (nullable = true)
+ |-- b: integer (nullable = true)
+ |-- c: string (nullable = true)
+ |-- d: string (nullable = true)
 
 
 -- !query 9
-DESC FORMATTED t PARTITION (c='Us', d=1)
+DESC t PARTITION (c='Us', d=1)
 -- !query 9 schema
 struct<col_name:string,data_type:string,comment:string>
 -- !query 9 output
-# Detailed Partition Information	                    	                    
+a                   	string              	                    
+b                   	int                 	                    
+c                   	string              	                    
+d                   	string              	                    
 # Partition Information	                    	                    
-# Storage Information	                    	                    
 # col_name          	data_type           	comment             
-Compressed:         	No                  	                    
-Database:           	default             	                    
-Location: sql/core/spark-warehouse/t/c=Us/d=1	                    
-Partition Parameters:	                    	                    
-Partition Value:    	[Us, 1]             	                    
-Storage Desc Parameters:	                    	                    
-Table:              	t                   	                    
+c                   	string              	                    
+d                   	string
+
+
+-- !query 10
+DESC EXTENDED t PARTITION (c='Us', d=1)
+-- !query 10 schema
+struct<col_name:string,data_type:string,comment:string>
+-- !query 10 output
 a                   	string              	                    
 b                   	int                 	                    
 c                   	string              	                    
+d                   	string              	                    
+# Partition Information	                    	                    
+# col_name          	data_type           	comment             
 c                   	string              	                    
 d                   	string              	                    
-d                   	string
+                    	                    	                    
+# Detailed Partition Information	                    	                    
+Database            	default             	                    
+Table               	t                   	                    
```

Comment:
not related to this PR, but why do we put the `Database` and `Table` in `# Detailed Partition Information`? Did we follow hive?